src/web/functions.php

```diff
@@ -11,7 +11,232 @@
  */
 function getUniqueFirstLetters(array $airports)
 {
-    // put your logic here
+    $letters = [];
+    foreach ($airports as $var) {
+        $letters[] = getFirstLetter($var);
+    }
+    $letters = array_unique($letters);
+    sort($letters);
 
-    return ['A', 'B', 'C'];
+    return $letters;
+}
+
+/**
+ * Returns first letter of airports array item name
+ *
+ * @param $var
+ * @return mixed
+ */
+function getFirstLetter($var)
+{
+
```

Comment:
not needed empty line before return if it is single line in block

Reply:
Updated